### DISCUSSION TITLE
No output from electron detector. What have I got wrong?

### DISCUSSION THREAD

**Discussion by AnnaSv** [ozel/DIY_particle_detector#40] [Q&A] [answered]
Hi, I have built up the electron detector version of this project and can't seem to get any signs of life out of it. The output is just noise of about 5mV on my oscilloscope and doesn't change in response to anything - exposing it to light doesn't change it, nor does placing a 10 uCi Cs-137 test source right next to it.  It actually appears to be the same amount of noise whether the battery is plugged in or not, but I do see a voltage spike if I unplug the battery. For testing, I'm wrapping the PCB in electrical tape then adhesive copper foil because I don't have a suitable box. I've double checked the components and soldering, I've got 9.6kOhms across the battery leads, 9V across V+ and V- of the amplifier... and I've made a different PIN radiation detector where the BPW34's are reverse biased and that one produces very clear ~200mV pulses next to the Cs-137, so I know my scope still works! Any corrections or thoughts on further debugging appreciated.
![PXL_20211026_183716851](https://user-images.githubusercontent.com/22248817/138943042-e3eb83e5-d3c3-4fb6-a552-a8dc1fbeb60d.jpg)
![PXL_20211026_163859874](https://user-images.githubusercontent.com/22248817/138943109-2aba9a6a-ba30-419f-8c8e-0f139349d79b.jpg)

**Comment by ozel** [comment, marked as answer]
Hi Anna,
thanks for the pictures and detailed report, really helpful for debugging. 
Your built looks perfect!
Even the SMD type BPW34S diodes seem to be mounted with the correct polarity.
You've probably seen the troubleshooting guide, right?
https://github.com/ozel/DIY_particle_detector/wiki/Troubleshooting

The most likely issue is that you are missing a metal box and your soft copper tape solution just isn't equivalent enough to a proper Faraday cage and complete light blocker.
If the noise is much lower than +/- 25 mV for the electron detector as in your case, it means the amplifier is oversaturated. Either with electromagnetic radiofrequency waves, light, or both. See the scope screenshots and explanations of point 9 in the guide.
You can really take any large enough candy tin box, the battery must go in there, too! Also don't forget the cable going to the scope needs to be shielded.
You can simplify the debug setup by throwing everything, **including the oscilloscope probe** inside the metal box. The probe cable may be routed simply through the slit between the box's lower part and its cover (just bend it slightly and recover it with some black cloth if light could sneak through there).
If you don't screw the detector board to the metal box walls, one ground wire should be soldered from the minus pins of the detector to the metal case.
Also, in that case, make sure there's no short circuit between the metal box and the detector board beside that ground wire. (this is likely an issue in your electric tape + copper tape setup, or it simply doesn't work due to capacitive coupling because the copper tape is too close to the amplifier and diodes or not properly grounded).

If the signal noise won't increase inside a metal box you have a component error, like a broken ceramic capacitor or the amplifier chip (typically gets hot in that case). But that is quite rare.

You could also tripple check the values/markings of all yellow ceramic capacitors (in particular C1 and C2) and R3. I've verified the other components via the pictures. 

I believe you're almost there, good luck.

**Comment by AnnaSv** [comment]
Thanks so much for the quick and detailed response! Of the three circuits that I had built up, I got one working immediately by putting it, and battery, and scope probe in a big metal box as you suggested. The noise floor looked about +/-30mV with pulses about -150mV to -200mV with the Cs137 sample in the box right next to the diodes. For this circuit I had used through-hole BPW34's instead of the SMT BPW34FS-Z's above, and a TL072 amplifier instead of a TLE2072 (the latter hadn't arrived in the mail yet). 
![PXL_20211027_191603182](https://user-images.githubusercontent.com/22248817/139164457-4ab9df93-7776-45cd-ac32-b4e5755215bd.jpg)

So I read the datasheets and I guess the sensitivity of the BPW34 is a bit better than that SMT version, and that TLE2072 should be less noisy than the TL072. So I did some reworking to construct a circuit that *actually* uses all the recommended components, and wouldn't you know, noise floor is a bit lower and pulses are a bit larger: 
![PXL_20211027_235203890](https://user-images.githubusercontent.com/22248817/139164018-fb6ac070-8563-42c4-906c-e380948ad84c.jpg)
It's also more sensitive to any ambient light, which is why it's wearing my jacket. :)

**Reply by ozel** [reply]
congratulations, good job!

the TLE2072 is quite a bit better in signal-to-noise ratio for this application, which is important the more diodes are connected. Since you are quite versed in soldering, you can try to double them up: solder on more BPW34 on top of each. I've never had time to try that properly. Noise should increase but the advantage of a doubled sensor volume may be dominating (= more triggers per time). Please let me know the outcome if you try it.

The type of BPW34 does not affect the detection of ionizing radiation / subatomic particles - all types seem to carry the same silicon chip (I compared some in https://www.mdpi.com/1424-8220/19/19/4264). 
Actually, there is zero sensor info in the original datasheets relevant for this detector application. :-)
However, the diodes with a transparent casing are obviously more susceptible to ambient light! From that perspective, the SMT style BPW34FS-Z you had first is actually superior to the plain BPW34 in the transparent DIP package.

I usually try to get the cheapest BPW34 type available versus the level of light-shielding that I am willing to do. In beginner workshops with students, I always got for the black epoxy-covered DIP package F-types (BPW34F or BPW34FA), even if they are slightly more expensive sometimes.
